docs/source/en/model_doc/clip.mdx

```diff
@@ -75,6 +75,33 @@ encode the text and prepare the images. The following example shows how to get t
 
 This model was contributed by [valhalla](https://huggingface.co/valhalla). The original code can be found [here](https://github.com/openai/CLIP).
 
+## Resources
+
+A list of official HuggingFace and community (indicated by 🌎) resources to help you get started with CLIP. If you're
+interested in submitting a resource to be included here, please feel free to open a Pull Request and we will review it.
+The resource should ideally demonstrate something new instead of duplicating an existing resource.
+
+<PipelineTag pipeline="text-to-image"/>
+- A blog post on [How to use CLIP to retrieve images from text](https://huggingface.co/blog/fine-tune-clip-rsicd)
+- A blog bost on [How to use CLIP for Japanese text to image generation](https://huggingface.co/blog/japanese-stable-diffusion)
+- A Huggingface space on [Finding best reaction gifs to a given text using CLIP](https://huggingface.co/spaces/flax-community/clip-reply-demo)
+
+<PipelineTag pipeline="image-to-text"/>
+- A Huggingface space on [Finding a prompt based on the image](https://huggingface.co/spaces/pharma/CLIP-Interrogator)
+- A Huggingface space showing [Guided diffusion using CLIP](https://huggingface.co/spaces/EleutherAI/clip-guided-diffusion)
+- A notebook showing [Video to text matching with CLIP for videos](https://github.com/NielsRogge/Transformers-Tutorials/blob/2d80e7293ccc417cd4f8f4fcca21ebcbda8f5d8f/X-CLIP/Video_text_matching_with_X_CLIP.ipynb)
+
+<PipelineTag pipeline="zero-shot-classification"/>
+- A notebook showing [Zero shot video classification using CLIP for video](https://github.com/NielsRogge/Transformers-Tutorials/blob/2d80e7293ccc417cd4f8f4fcca21ebcbda8f5d8f/X-CLIP/Zero_shot_classify_a_YouTube_video_with_X_CLIP.ipynb)
+
+
+🚀 Deploy
```

Comment:
I'd remove the deploy section entirely since none of these blog posts are specific to deploying CLIP.